### ISSUE TITLE
Narrator focus goes to back button after taking extra two tab navigation instead of directly moving to back button using touch gesture.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/WinUI-Gallery#381]
**Describe the bug**

**Steps to reproduce the bug**

Pre-requisite: 
- Turn on Narrator
- Set Narrator in Item Mode.
- Required touch enabled Device.

1. Launch XAML Controls Gallery.
2. Navigate to "All Controls" menu item and activate it.
3. Navigate to "Animated Visual Player" button and activate it.
4. Navigate through whole page using item navigation (Using one finger swipe) upto last item.
5. Try navigating to Back button using forward navigation(Using triple finger swipe) and observe the behavior.

**Expected behavior**

User should be able to navigate to back button.

**Actual behavior**

Narrator focus goes to back button after taking extra two tab navigation instead of directly moving to back button using touch gesture.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/76575839-9c0cde00-647d-11ea-9efa-1f47f49bc59b.png)

**Version Info**

Gallery version (found on Settings page): 

1.2.13.0

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/25441024)

**Comment by karkarl** [comment]
I am able to navigate to back button using triple finger swipe. Closing as no longer repros.
